### PR TITLE
[merged] Add bash completions

### PR DESCRIPTION
**Pull request by rhatdan**
This is just the beginning of a framework for bash completions.
Current completions just give you the list of options available.

Signed-off-by: Dan Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by alexlarsson** [comment]
@cgwalters-bot r+ 68806bf

**Comment by alexlarsson** [comment]
Lets try again with the warning fixed

@cgwalters-bot r+ 68806bf

**Comment by alexlarsson** [comment]
@cgwalters-bot retry

File: completions/base/bwrap [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```
Comment (alexlarsson) [inline completions/base/bwrap]: Seems this is called completions/base/bwrap instead of completions/bash/bwrap

**Comment by rhatdan** [comment]
@cgwalters-bot r+  d935ac6

**Comment by rhatdan** [comment]
@alexlarsson I don't think cgwalters-bot likes me.  Fixed the naming issue.

**Comment by alexlarsson** [comment]
@cgwalters-bot retry

**Comment by alexlarsson** [comment]
@cgwalters-bot r+  	d935ac6

**Pull request closed without merging**